src/cpp/server/server_cc.cc

```diff
@@ -106,43 +106,213 @@ class UnimplementedAsyncRequestContext {
 
 }  // namespace
 
+ServerInterface::BaseAsyncRequest::BaseAsyncRequest(
```

Comment:
Maybe try not to move this block.

Reply:
class ServerInterface is still in grpc namespace and class Server is moved to grpc_impl namespace. That is what is causing this movement as we have the grpc namespace declared above grpc_impl namespace and hence this move comes as a part of it. Does this make sense? Is it OK to leave it as-is now or should I move the grpc namespace below? (The diff doesn't show this change, but it is visible in line #276 on the new file on right hand side of diff).